istio/codescene-gateway.yaml

```
@@ -6,20 +6,20 @@ spec:
   selector:
     istio: ingressgateway # use istio default controller
   servers:
-  - port:
+  - hosts:
+    - "codescene.k8s-pcconsultants.de"
+    port:
       number: 80
       name: http
       protocol: HTTP
-    hosts:
-      - "<your.fqdn>"
     tls:
-      httpsRedirect: true # sends 301 redirect for http requests
-  - port:
+      httpsRedirect: false # sends 301 redirect for http requests
+  - hosts:
+    - "codescene.k8s-pcconsultants.de"
```

This is a publicly accessible instance which may be a security concern.
Moreover, it's accessible via HTTP.
Are you sure you want to disclose it (here)?

---

It is a dead site used only for training and no longer publicly accessible. 
```suggestion
    - "codescene.k8s-pcconsultants.de"
```
It is removed in next pull request.